OpenRA.Mods.Common/Traits/BotModules/CaptureManagerBotModule.cs

```diff
@@ -0,0 +1,194 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2018 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using OpenRA.Mods.Common.AI;
+using OpenRA.Support;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.Traits
+{
+	[Desc("Manages AI capturing logic.")]
+	public class CaptureManagerBotModuleInfo : ConditionalTraitInfo
+	{
+		[Desc("Actor types that can capture other actors (via `Captures` or `ExternalCaptures`).",
```

Comment:
`ExternalCaptures` doesn't exist anymore. Can you please amend this description?